### ISSUE TITLE
Deploy nodes in parallel

### ISSUE THREAD

**Issue by saschagrunert** [SUSE/skuba#1038]
Hey 👋,

I’m playing around with skuba and noticed that it would be useful for me to deploy nodes in parallel to safe some time. When speaking about worker nodes then it should be possible to use a classic work stealing approach for example. What do you think?

**Comment by evrardjp** [comment]
I guess this is only intended for worker nodes?
What prevents us from doing so in the current model?
I think you can run skuba join in parallel...

**Comment by saschagrunert** [comment]
> I guess this is only intended for worker nodes?

I think so, or adding masters / workers if the initial master already got deployed.

> What prevents us from doing so in the current model?

Nothing I guess. :man_shrugging: 

> I think you can run skuba join in parallel with a little bit of scripting, which means we probably shouldn't implement this in skuba.

Yes, I can do 
```bash
for ((i = 0; i < NODES; i++)); do
    skuba node join … &
done
wait
```

Feel free to close if you think it's out of scope.

**Comment by evrardjp** [comment]
yes I am using about the same scripting. I would think it's out of scope indeed, but I will let others decide.
